src/main/resources/hudson/plugins/copyartifact/CopyArtifact/config_de.properties

```diff
@@ -4,4 +4,6 @@ Artifacts\ to\ copy=Zu kopierende Artefakte
 Target\ directory=Zielverzeichnis
 Flatten\ directories=Verzeichnisse reduzieren
 Fingerprint\ Artifacts=Fingerabdr\u00fccke von Artefakten erstellen
-Optional=Optional
\ No newline at end of file
+Optional=Optional
+Fingerprint\ Artifacts=
+Include\ Build\ Number=
```

Comment:
This will translate the strings to an empty string. Almost certainly not what you want.

Reply:
Thanks @daniel-beck .  I saw that in the code review and wondered about it.  When I searched through Jenkins core and saw a number of message properties that were assigned an empty value, I thought that it must be a pattern that caused the property to be assigned the value from the default language.

An example:

https://github.com/jenkinsci/jenkins/blob/a530a99926940cb37fd2f7b66e5c2f458254a710/core/src/main/resources/hudson/logging/LogRecorder/configure_fr.properties#L24

Is the preferred pattern to leave the untranslated key out of the localized property file or is there a better way to handle it?

In this case, the best way to handle it may be to transition this plugin to use CrowdIn for its localization.  That will allow translators to submit messages through the CrowdIn user interface rather than directly editing property files.

@allancth would you like to submit the pull request to remove the untranslated properties or would you prefer that I do it so that you can approve the change?

Reply:
@allancth I submitted:

* https://github.com/jenkinsci/copyartifact-plugin/pull/171

If you approve and merge, it should result in a new release with a changelog that includes the bug fix.